lib/node/pl-middle-layer/src/network_check/network_check.test.ts

```diff
@@ -8,7 +8,7 @@ import { testCredentials } from './test_utils';
 // - locally
 // - stop minio and retest locally
 // - on demo server
-test('should return network connectivity report for a platform endpoint set in PL_ADDRESS env', async () => {
+test.skip('should return network connectivity report for a platform endpoint set in PL_ADDRESS env', async () => {
```

Comment:
Why?

Reply:
It's 80 seconds test with no asserts, it's just to see what is the output of the network check tool. It should be skipped, however I forgot to do it in the previous PR.